test/ng/compileSpec.js

```diff
@@ -4259,6 +4259,78 @@ describe('$compile', function() {
           });
 
 
+          it('should trigger `$onChanges` for literal expressions when expression input value changes (simple value)', function() {
```

Comment:
Can you also add tests verifying that `$onChanges` is not called when a property of the input value changes?